packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -84,6 +84,48 @@ const getInfo = async (request: Request<MessageTypes.GetInfo>) => {
     } as const;
 };
 
+const estimateFee = async (request: Request<MessageTypes.EstimateFee>) => {
+    const api = await request.connect();
+
+    // There is currently no way of getting fees without a transaction, so we construct a dummy transaction to query for.
```

Comment:
I have a feeling that we do not need to do this and we can estimate the correct transaction, lets discuss this over call